.gitignore

```diff
@@ -4,7 +4,7 @@
 # Eclipse, Netbeans and IntelliJ files
 /.*
 !/.github
-!/.jenkins
+!/.ci
```

Comment:
I presume this file needs this change too over at optaplanner-quickstarts?
  https://github.com/kiegroup/optaplanner-quickstarts/blob/stable/.gitignore
If it does, would you mind making a PR for that too?